### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by burtbeckwith**

### CODE REVIEW

File: object-storage-aws/src/main/java/io/micronaut/objectstorage/aws/AwsS3Operations.java [hunk 1]
```diff
@@ -17,11 +17,21 @@
 
 import io.micronaut.context.annotation.EachBean;
 import io.micronaut.context.annotation.Parameter;
-import io.micronaut.objectstorage.*;
```
Comment (alvarosanchez) [inline object-storage-aws/src/main/java/io/micronaut/objectstorage/aws/AwsS3Operations.java:20]: I use IntelliJ import optimization. I wonder whether we should persist this via `.editorconfig` or similar. Otherwise, IDEA may want to revert your change the next time I change the file.

File: object-storage-aws/src/test/groovy/io/micronaut/objectstorage/aws/AbstractAwsS3Spec.groovy [hunk 2]
```diff
@@ -9,11 +9,12 @@ import software.amazon.awssdk.services.s3.S3Client
...
 import software.amazon.awssdk.services.s3.model.DeleteBucketRequest
 
+import static io.micronaut.objectstorage.aws.AwsS3Configuration.PREFIX
+
 abstract class AbstractAwsS3Spec extends ObjectStorageOperationsSpecification implements TestPropertyProvider {
 
     public static final String BUCKET_NAME = System.currentTimeMillis()
-
-    public static final String OBJECT_STORAGE_NAME = "default"
+    public static final String OBJECT_STORAGE_NAME = 'default'
```
Comment (alvarosanchez) [inline object-storage-aws/src/test/groovy/io/micronaut/objectstorage/aws/AbstractAwsS3Spec.groovy:17]: For my own benefit: what's the rationale behind this change? If I'm not mistaken, in both cases the type will be `java.lang.String`. Is it the single quoted variant faster?

**Review by alvarosanchez — APPROVED** [review]
I ❤️  your cleanup PRs, I missed them! Thank you!

File: object-storage-aws/src/test/groovy/io/micronaut/objectstorage/aws/AbstractAwsS3Spec.groovy (hunk 2 above)
Comment (burtbeckwith) [reply object-storage-aws/src/test/groovy/io/micronaut/objectstorage/aws/AbstractAwsS3Spec.groovy:17]: My preference is to use single quotes for Strings to make it clear that it's not a GString. No performance difference

File: object-storage-aws/src/main/java/io/micronaut/objectstorage/aws/AwsS3Operations.java (hunk 1 above)
Comment (burtbeckwith) [reply object-storage-aws/src/main/java/io/micronaut/objectstorage/aws/AwsS3Operations.java:20]: You can configure the number of imports allowed before grouping with * in IntelliJ settings Editor->Code Style->Java->Imports - I changed the values to 100 and checked "Use single class import"
